rpc_jobs/release.yml

```diff
@@ -0,0 +1,101 @@
+- job:
+    name: RE-Release
+    project-type: workflow
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - rpc_gating_params
+      - string:
+          name: RELEASE_VERSION
+          description: |
+            The version that will be tagged by this job
+      - string:
+          name: NEXT_VERSION
+          description: |
+            The version that will be released after the next stabilisation
+            period.
+      - string:
+          name: MAINLINE_BRANCH
+          description: The MAINLINE_BRANCH branch to work from
+          default: master
```

Comment:
I'd rather not see a default here, everything else needs to be specified anyway. Lets force the user to provide them all to help prevent mistakes, in an automated world this strikes me as the kind of thing that would be easy to slip through.

Reply:
Done